CONTRIBUTING.md

```diff
@@ -179,30 +179,33 @@ config.ichain_force_test_attributes = {:email => "testuser@example.com"}
 2. **Design**
   * Related to the looks and/or usability of the application; needs attention from someone who understands front-end and UX
   * If you are good with graphics and design, give it a shot!
-3. **Documentation**
+3. **DO NOT MERGE**
```

Comment:
I don't think we really need numbering here. Maybe you could just replace all the `N.` with `*`.